erpnext/patches/v7_1/set_total_amount_currency_in_je.py

```diff
@@ -0,0 +1,28 @@
+import frappe
+from frappe.utils import cint
+
+def execute():
+	for je in frappe.get_all("Journal Entry", fields=["name", "multi_currency"]):
```

Comment:
above for loop, set default company currency if not multi-currency and pay_to_recd_from has value